packages/telemetry/src/events/session.ts

```diff
@@ -125,6 +129,10 @@ export function eventCliSession(
 		// Config Values
 		config: configValues,
 		flags: cliFlags,
+		// Total Number of Integrations
+		totalIntegrations: userConfig?.integrations?.length,
```

Comment:
We don't need this field as Mixpanel can give us this in its reporting. We've avoided during this sort of pre-processing on purpose.